14_2_Using_i2p.md

```diff
@@ -0,0 +1,50 @@
+# Chapter 14.2: Using i2p
+
+The Invisible Internet Project (I2P) is a fully encrypted private network layer. It uses a distributed [network database](https://geti2p.net/en/docs/how/network-database) and encrypted unidirectional tunnels between your and your peers.
+
+Basic differences between Tor and i2p: https://geti2p.net/en/comparison/tor
+
+Trade-offs: https://bitcoin.stackexchange.com/questions/107060/tor-and-i2p-tradeoffs-in-bitcoin-core
+
+It is not installed by [Bitcoin Standup](https://github.com/BlockchainCommons/Bitcoin-Standup-Scripts) right now as i2p support was recently added in Bitcoin core. However, you can try it manually by following the below steps:
+
+1. Install `i2pd`: https://i2pd.readthedocs.io/en/latest/user-guide/install/
+2. Run `i2pd`: [`sudo systemctl start i2pd.service`](https://i2pd.readthedocs.io/en/latest/user-guide/run/)
+3. Confirm if `i2pd` is running: `ss -nlt`
+
```

Comment:
For commands, let's stick with the of showing the command-line options.

So, you'd have something like this:

You can install `i2pd` with the [installation instructions](URL).

Then, run it and check that it's running:
```
# sudo systemctl start i2pd.service
# ss -nlt
```

Reply:
Made changes in https://github.com/BlockchainCommons/Learning-Bitcoin-from-the-Command-Line/commit/e6faf1cc07f4477c6508bbf82b1dc59940f101d9